### DISCUSSION TITLE
To upgrade or not to upgrade to v1?

### DISCUSSION THREAD

**Discussion by xenide** [wevm/wagmi#2824] [Q&A]
To the devs who have upgraded to the latest release (v1), what are the improvements over the 0.12.* versions? 

Are there actually performance improvements or new features? Cuz it's quite hard to tell on my end.

Thanks!

**Comment by jxom** [comment]
Improvements mainly come from the migration of ethers to viem. See the rationale: https://viem.sh/docs/introduction.html + benchmarks: https://viem.sh/docs/benchmarks.html

some testimonials:
https://twitter.com/boilingsoupdev/status/1675673785373175808
https://twitter.com/0x_insuline/status/1674025132757590021
https://twitter.com/jakemoxey/status/1673846338067529730
https://twitter.com/varunsrin/status/1673539447097532416
